packages/wonder-blocks-typography/src/util/styles.ts

```diff
@@ -49,8 +49,8 @@ const styles: StyleDeclaration = StyleSheet.create({
             lineHeight: font.lineHeight.xLarge,
         },
         [breakpoint.mediaQuery.mdOrSmaller]: {
-            fontSize: font.size.large,
-            lineHeight: font.lineHeight.large,
+            fontSize: "22px",
+            lineHeight: "26px",
```

Comment:
Now that I think, this could be better left as rem unit (still one off for now) so it is consistent with the other new rem tokens? wdyt?

Sorry for going back and forth with this.

Reply:
Oh yeah, that totally makes sense! Thank you for the suggestion. I'll update it!